### PR TITLE
Create interview-lana-brindley.rst

### PR DESCRIPTION
**Pull request by Geodun**
This an interview for the community spotlight section of the hiring and getting hired guide.

### CODE REVIEW

File: docs/hiring-guide/community-spotlight/interview-lana-brindley.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Interview with Lana Brindley
+=========================
+
+Lana is a technical writer at SUSE. Her work has primarily been with open source software, and she has been writing technical documentation and leading writing teams for 12 years.
+
+Key takeaways
+*************
+*Write whatever you can to build and show off your skills.
```
Comment (plaindocs) [inline docs/hiring-guide/community-spotlight/interview-lana-brindley.rst]: These probably need a line before the heading, and a space after the asterisk to display properly.

File: docs/hiring-guide/community-spotlight/interview-lana-brindley.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+As for contributing to open source, I know that documentation is often held up as a good way to get started, but I can appreciate that it's not always easy. That said, most larger open source groups (especially those that are backed by organisations like SUSE and Red Hat) have some kind of mentoring program that can help you with your first contribution if you really don't know where to start. And that's probably my other big tip: no matter where you are in your career, always choose at least one mentor and one mentee. There is always something you can learn, and something you can teach
+
+Lana goes by ''loquacity'' on the Write the Docs Slack group, and @Loquacities on Twitter. You can find her in the ''#australia'', ''#managing-writers'', and ''#remote channels''.
```
Comment (plaindocs) [inline docs/hiring-guide/community-spotlight/interview-lana-brindley.rst]: These double quotes need to be double back ticks

```rst
    ``this``
    ''not this''
```

**Review by plaindocs — CHANGES_REQUESTED** [review]
Other than those two things, this looks great!

You can always see how a commit will look by clicking the Netlify preview:

image

**Comment by plaindocs** [comment]
I think fixing the bullet points will also fix the Travis CI warning.

File: docs/hiring-guide/community-spotlight/interview-lana-brindley.rst (hunk 2 above)
Comment (Geodun) [reply docs/hiring-guide/community-spotlight/interview-lana-brindley.rst]: Fixed, thank you.

File: docs/hiring-guide/community-spotlight/interview-lana-brindley.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    writer your resume is a writing sample. Make sure that you show that you can write prose as well as dot points, and be really
+    mindful of the words you choose, and the layout and design.
+
+    As for contributing to open source, I know that documentation is often held up as a good way to get started, but I can appreciate
+    that it's not always easy. That said, most larger open source groups (especially those that are backed by organisations like SUSE
+    and Red Hat) have some kind of mentoring program that can help you with your first contribution if you really don't know where to
+    start. And that's probably my other big tip: no matter where you are in your career, always choose at least one mentor and one
+    mentee. There is always something you can learn, and something you can teach
+
+Lana goes by ``loquacity`` on the Write the Docs Slack group, and @Loquacities on Twitter. You can find her in the ``#australia``, ``#managing-writers``, and ``#remote channels``.
```
Comment (plaindocs) [inline docs/hiring-guide/community-spotlight/interview-lana-brindley.rst]:
```suggestion
Lana goes by ``loquacity`` on the Write the Docs Slack group, and @Loquacities on Twitter. You can find her in the ``#australia``, ``#managing-writers``, and ``#remote`` channels.
```

**Review by plaindocs — APPROVED** [review]

**Pull request merged**